test/DynamoCoreTests/CallsiteTests.cs

```diff
@@ -201,10 +196,10 @@ public void Callsite_ElementBinding_CustomNodes_multiple()
             Assert.IsFalse(ws.Nodes.OfType<Function>().First().IsInErrorState);
 
             BeginRun();
-            AssertPreviewValue("da39dbe5f59649b18c2fb6ca54acba7b", 111111);
-            AssertPreviewValue("2366239164a9441a8c4dcd981d9cf542", 222222);
-            AssertPreviewValue("8cfce012280342f3bd688520d68a7f66", 333333);
-            AssertPreviewValue("08448232ee094aad8280e9a99ed44f46", 444444);
+            AssertPreviewValue("da39dbe5f59649b18c2fb6ca54acba7b", 6);
+            AssertPreviewValue("2366239164a9441a8c4dcd981d9cf542", 7);
+            AssertPreviewValue("8cfce012280342f3bd688520d68a7f66", 10);
+            AssertPreviewValue("08448232ee094aad8280e9a99ed44f46", 11);
```

Comment:
Sorry, what are these values that have changed?

Reply:
handwritten values were replace with values generated by interacting with the graph instead. It's a bit harder to write values directly into trace by hand since the data is bas64 encoded and gzipped